<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserHomeForm</class>
 <widget class="QMainWindow" name="UserHomeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户中心</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="cnterD" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>271</width>
       <height>108</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="lblDPuzzleMade">
        <property name="text">
         <string>出题数目</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblDName">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblDLevel">
        <property name="text">
         <string>等级</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblDTag">
        <property name="text">
         <string>个性签名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtDName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtDTag">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtDPuzzleMade">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtDLevel">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="cnterP" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="lblPExp">
        <property name="text">
         <string>经验值</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtPName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblPLevel">
        <property name="text">
         <string>等级</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblPName">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblPuzzlePassed">
        <property name="text">
         <string>闯关数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblPTag">
        <property name="text">
         <string>个性签名</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtPTag">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtPPuzzlePassed">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtPExp">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtPLevel">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnAddWord">
         <property name="text">
          <string>出题</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPlay">
         <property name="text">
          <string>开始游戏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAll">
         <property name="text">
          <string>所有用户</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnEditName">
         <property name="text">
          <string>修改姓名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEditTag">
         <property name="text">
          <string>修改个性签名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLogout">
         <property name="text">
          <string>退出登录</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>79</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="lblUserName">
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblType">
       <property name="text">
        <string>用户类别</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtUserName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtType">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>cnterP</zorder>
   <zorder>cnterD</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>gridLayoutWidget_3</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnAddWord</tabstop>
  <tabstop>btnPlay</tabstop>
  <tabstop>btnAll</tabstop>
  <tabstop>btnEditName</tabstop>
  <tabstop>btnEditTag</tabstop>
  <tabstop>btnLogout</tabstop>
  <tabstop>txtPName</tabstop>
  <tabstop>txtPTag</tabstop>
  <tabstop>txtPPuzzlePassed</tabstop>
  <tabstop>txtPExp</tabstop>
  <tabstop>txtPLevel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
